src/main/java/com/yolo/customer/recipe/recipeImage/RecipeImage.java

```diff
@@ -0,0 +1,39 @@
+package com.yolo.customer.recipe.recipeImage;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.time.LocalDateTime;
+
+@Getter
+@Setter
+@Entity
```

Comment:
specify table name on with @Entity that corresponds to table name in db as it might cause error while testing